planet/src/main/webapp/css/profile.css

```diff
@@ -1,5 +1,5 @@
 .profile-container {
-    width: min(65%, 600px);
+    width: min(70%, 800px);
```

Comment:
We might want to consider using vh, vw or em instead of px to make this responsive!

Reply:
It is, actually, responsive since I am using percentages. As for the pixels, they just represent a maximum limit of the width. In other words, If the screen's width is larger than 1143 px, the width of the container shouldn't be more than 800px! But if it's smaller, then, it should take 70% of the width.